MediaBrowser.Model/Subtitles/FontFile.cs

```diff
@@ -0,0 +1,34 @@
+#nullable disable
+#pragma warning disable CS1591
```

Comment:
Should both be enabled for new files

Reply:
It will run into this error.
`Subtitles\FontFile.cs(11,23): error CS8618 Indicate that a non-nullable Field must be initialized where the type is constructed`

Reply:
Make it nullable or set a default value (`string.Empty`)